.github/CODEOWNERS

```diff
@@ -580,6 +580,9 @@ Makefile                                              @thiagokokada
 /modules/targets/darwin                               @midchildan
 /tests/modules/targets-darwin                         @midchildan
 
+/modules/programs/wofi.nix                            @christoph-heiss
+/tests/modules/programs/wofi                          @christoph-heiss
+
```

Comment:
Oh, just realized this is a program not a service. It should go up with the programs around line 350